ISSUE_TEMPLATE/component_validation_checklist.md

```diff
@@ -0,0 +1,61 @@
+# Component validation checklist
+
+Here is a checklist of items that need to be completed before a component or pattern can be marked as Ready in the Design System.
+
+- [ ] Documentation is complete. [Documentation writing info](#)
+    - Read through documentation and make sure the following is done
+      - [ ] Description of component or pattern
+      - [ ] HTML code snippet example(s)
+      - [ ] React code snippet example(s)
+      - [ ] All react properties are explained
+      - [ ] Guidance on when to use this item
+      - [ ] Guidance on when not to use this item
+      - [ ] Accessibility considerations
+
+---
+
+- [ ] Validate HTML
+  - Go to [validator.w3.org](https://validator.w3.org/#validate_by_input) paste your code in and make sure it validates without error.
+
+---
+
+- [ ] Keyboard testing
+  - Navigate the item using your Keyboard
+  - What to check:
+    - [ ] Navigate using **tab (navigate forward)** and **shift + tab (navigate backward)**
+      - Keyboard focus indicators are present
+      - Navigation order is logical and intuitive
+    - [ ] Links - Follow links by using **enter**
+    - [ ] Buttons - use **enter** or **spacebar** to click a button
+    - [ ] Checkbox - **spacebar** to check/uncheck
+    - [ ] Radio buttons - **up/down** arrows or **left/right** arrows to select an option
+    - [ ] Dropdowns - **up/down** arrows to navigate between options. **Spacebar** to open close select
+    - [ ] Autocomplete - **Type** to begin filtering. **up/down** arrows to navigate between options. **Enter** to select an option.
+    - [ ] Dialog/modal - **Esc** should close the window
+      - Modal dialogs should maintain keyboard focus.
+      - When a dialog closes, focus should usually return to the element that opened the dialog.
+    - [ ] Tabs - **Tab** once to navigate into the group of tabs and once to navigate out of the group of tabs. **up/down** arrows or **left/right** arrows to move through tabs.
```

Comment:
There's some debate on this one, about how Tabs should behave. We came down on the side of using ARIA attributes, and making tabs proper links that don't respond to arrow keys. Used this as our guidance: https://web.archive.org/web/20180724032341/https://simplyaccessible.com/article/danger-aria-tabs/

I opted for the Waback Machine b/c the SSL cert is expired just now.

Reply:
Can you re-write that `Tabs` check item and post here?

Reply:
☝️@1Copenut

Reply:
@line47 I can get to it, but it'll be at least a day, maybe not until Monday.

Reply:
Hey @1Copenut any chance you could update this?